pkg/machine/e2e/config.go

```diff
@@ -156,8 +156,10 @@ func runWrapper(podmanBinary string, cmdArgs []string, timeout time.Duration) (*
 		return nil, err
 	}
 	ms := machineSession{session}
-	ms.waitWithTimeout(timeout)
-	fmt.Println("output:", ms.outputToString())
+	if wait {
+		ms.waitWithTimeout(timeout)
+		fmt.Println("output:", ms.outputToString())
+	}
```

Comment:
Not a hold up for this PR, but could we make the timeout adjustable by the user rather than hard coding it to 90 seconds?

Reply:
I think that is the standard for integration. Would the require an arg to the run function?